.github/workflows/release-notes.yaml

```
@@ -0,0 +1,72 @@
+#
+# Copyright (C) 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: release-notes
+
+on:
+  workflow_dispatch:
+    inputs:
+      milestone:
+        description: 'Milestone to generate release notes from'
+        required: true
+env:
+  GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+jobs:
+
+  build:
+    name: Generate release notes
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```
needs to be to a fixed version to avoid surprise on updates